### PR TITLE
Outline for the Prometheus README

### PR DESCRIPTION
**Pull request by jasonBirchall**
**WHAT**
An outline of the Prometheus install documentation. 

**WHY**
As you can probably tell, the document is not complete. This PR is just to outline the intention of the doc and a chance for anyone to comment on the structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+
+This repository will allow you to create a monitoring namespace in a MoJ Cloud Platform cluster. It will also contain the neseccary values to perform an installation of Prometheus-Operator and Kube-Prometheus. 
+
+  - [Pre-reqs](#prereq)
+  - [Creating a monitoring namespace](#installation)
+  - [Installing Prometheus-Operator](#output-example)
+  - [Installng Kube-Prometheus](#usage)
+  - [Exposing the port](#features-and-advantages-of-this-project)  
+  - [How to tear it all down](#am-i-missing-some-essential-feature)
+  - [Contributing](#contributing)
```
Comment (kalbir) [inline README.md]: Looks like a good set of steps but I'd also say -- this is for someone else on the team right now so you can probably rely on the code a bit and just point people to the things they need to add/change/manage.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+TL;DR
+# Copy dir to your namespace in the cloud-platform-environments repository.
+
+# Add CoreOS Helm repo
+$ helm repo add coreos https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
+
+# Install prometheus-operator
+$ helm install coreos/prometheus-operator --name prometheus-operator --namespace monitoring -f ./helm/prometheus-operator/values.yaml
+
+# Install kube-prometheus
```
Comment (kalbir) [inline README.md]: As discussed, I'm not sure that we need to do this as we can set up everything via the operator. Would be good to see where you end up with it.

File: README.md (hunk 1 above)
Comment (jasonBirchall) [reply README.md]: Really good point. Will be sure to take that into consideration going forward.

File: README.md (hunk 2 above)
Comment (jasonBirchall) [reply README.md]: Yes, will have a chat to Mitch about this.

**Pull request merged**